src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -856,6 +859,10 @@ void ImageReplayer<I>::set_mirror_image_status_update(
     ceph_assert(!"invalid state");
   }
 
+  if (m_perf_counters != nullptr){
+    m_perf_counters->set(l_rbd_mirror_image_status, status.state);
```

Comment:
I'm not sure I see the point of this `image_status` counter.  What value does it bring to the user?  If rbd-mirror daemon isn't doing well, the image can report a "happy" status and yet still be hours behind.  The metric that user is interested in is whether the image is caught up, if the image is not caught up, whether and how fast it's catching up, etc.

The image's status should be mostly irrelevant, particularly the way it's reported here -- as in integer.  Who is going to map `5` to `MIRROR_IMAGE_STATUS_STATE_STOPPING_REPLAY`?

Reply:
@idryomov , you are right, the image status metric is for inform about what you commented:
_"The metric that user is interested in is whether the image is caught up, if the image is not caught up, whether and how fast it's catching up, etc."_

We thought, that `status.state` provided that information. If this variable does not provide the right information, what is the right source to use?